cmd/dlt/accountroles/cmd.go

```diff
@@ -151,25 +151,53 @@ func run(cmd *cobra.Command, _ []string) {
 			reporter.Errorf("Error getting role: %s", err)
 			os.Exit(1)
 		}
+		if role.RoleARN == "" {
+			reporter.Errorf("There are no roles to be deleted")
+			os.Exit(1)
+		}
 		if !checkIfRoleExists(clusters, role) {
 			finalRoleList = append(finalRoleList, role.RoleName)
+		} else {
+			reporter.Errorf("Please make sure there are no clusters associated with role %s", role.RoleName)
+			os.Exit(1)
```

Comment:
TBH, I think we should eliminate the `--role-name` flow and use only `--prefix`. For one, when we create roles with `rosa create account-roles` we do so in a bundle rather than individual roles, a user manually creating things with their own naming scheme will also not be using `rosa` to delete them. Second, if I use `--role-name` and delete the installer role, then I go and do the `--prefix` flow, it no longer finds _any_ of the roles, since they are searched based on the installer role. I think this flow is prone to misuse by users.

Reply:
ok removed the role name flow